src/Stream/src/StreamBase/Binder/PartitionHandler.cs

```diff
@@ -35,15 +37,51 @@ public class PartitionHandler
 
         public int PartitionCount { get; set; }
 
+        public IExpressionParser ExpressionParser
+        {
+            get
+            {
+                if (_expressionParser != null)
```

Comment:
Is this correct, or should it be `== null`?

Reply:
Fixed